apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.html

```diff
@@ -1,8 +1,8 @@
-<perun-web-apps-immediate-filter (filter)="applyFilter($event)" [placeholder]="'MEMBER_DETAIL.SETTINGS.RESOURCE_ATTRIBUTES.FILTER' | translate"></perun-web-apps-immediate-filter>
-
+<h1 class="page-subtitle">{{getTitle() | translate}}</h1>
```

Comment:
This is not working at all
![image](https://user-images.githubusercontent.com/32871167/94563210-a69a9380-0266-11eb-9db0-212dc0a7bda8.png)
The title could be for sure not in this component but in the "user-settings-facility-component"